ethclient/ethclient_test.go

```diff
@@ -223,15 +223,15 @@ func (m *mockHistoricalBackend) Call(ctx context.Context, args ethapi.Transactio
 	if ok && num == 100 {
 		return hexutil.Bytes("test"), nil
 	}
-	return nil, ethapi.ErrHeaderNotFound
```

Comment:
Maybe it's worth changing the `ethapi.ErrHeaderNotFound` definition to `fmt.Errorf("header %w", ethereum.NotFound)`, to avoid changing it everywhere one by one? Or does that break other tests?